v3/cypress/e2e/cfm.spec.ts

```diff
@@ -0,0 +1,33 @@
+import { CfmElements as cfm } from "../support/elements/cfm"
+
+context("CloudFileManager", () => {
+  beforeEach(function () {
+      const queryParams = "" // "?sample=mammals&dashboard&mouseSensor"
```

Comment:
You very probably left the commented out query params here for reference since they could potentially be useful to see as an example, but I'm noting them just in case you didn't mean to leave them here.